### ISSUE TITLE
Be sure all shortcuts have corresponding 'longcuts' in the menus

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/iD#6397]
The Keyboard shortcuts should all be real shortcuts.
I.e., they all should have corresponding "longcuts".
i.e., they should all be items already available in various menus, not standalone.
Not only would users be able to find them easier when looking among related items in the menus,
but that would make them available in cases when the shortcut was unusable (#5306).

**Comment by bhousel** [comment]
I agree that we should not over-rely on keyboard entry, for eventual touch/mobile device support.
We'll think about how to best do this over the next few months as we plan out iD v3.

**Comment by quincylvania** [comment]
I added toggle buttons for all the panels that were shortcut-only: Background, Location, History, Measurement. I also did #2508, so copy and paste of features can now be performed without a keyboard.

While there are still some shortcuts that don't correspond directly to UI controls (zoom in/out by a lot, nudge features with arrow keys, select first/last/next vertex in a way), these are all things users can essentially accomplish via other interactions.

We should continue to pursue feature parity no matter what combination of input devices the mapper is using. See also #7713.
